src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,32 @@
+package duke.command;
+
+import duke.History;
+import duke.ResponseFormatter;
+import duke.Storage;
+import duke.TaskList;
+
+public class ByeCommand extends Command {
+    public static final String COMMAND = "Bye!";
+
+    /**
+     * Constructor for ByeCommand
+     * will mark isExit as true so duke can escape while loop
```

Comment:
Perhaps you should add a full stop after every sentence.
I think your second sentence should start with a singular verb `Marks` as well.